Tests/FDBTests/TupleTests.swift

```diff
@@ -150,6 +150,60 @@ class TupleTests: XCTestCase {
         }
     }
 
+    func testFloat() throws {
+        let cases: [(Float32, Bytes)] = [
+            (-10000.01, [32, 57, 227, 191, 245]),
```

Comment:
Test cases generated using Java client (code for history, needs converting from Int8 to UInt8):

```java
import java.nio.ByteBuffer;
import java.nio.ByteOrder;
import java.nio.charset.StandardCharsets;
import java.util.Arrays;

class Main {
  private static final byte FLOAT_CODE = 0x20;
  private static final byte DOUBLE_CODE = 0x21;
  private final static char[] hexArray = "0123456789ABCDEF".toCharArray();

  public static String bytesToHex(byte[] bytes) {
      char[] hexChars = new char[bytes.length * 2];
      for ( int j = 0; j < bytes.length; j++ ) {
          int v = bytes[j] & 0xFF;
          hexChars[j * 2] = hexArray[v >>> 4];
          hexChars[j * 2 + 1] = hexArray[v & 0x0F];
      }
      return new String(hexChars);
  }

  static byte[] floatingPointCoding(byte[] bytes, int start, boolean encode) {
		if(encode && (bytes[start] & (byte)0x80) != (byte)0x00) {
			for(int i = start; i < bytes.length; i++) {
				bytes[i] = (byte) (bytes[i] ^ 0xff);
			}
		} else if(!encode && (bytes[start] & (byte)0x80) != (byte)0x80) {
			for(int i = start; i < bytes.length; i++) {
				bytes[i] = (byte) (bytes[i] ^ 0xff);
			}
		} else {
			bytes[start] = (byte) (0x80 ^ bytes[start]);
		}

		return bytes;
	}

  public static void main(String[] args) {
    Float[] array = {-10000.01f, -6500.1234f, -100.00001f, -1f, 0f, -0f, 1f, 1.1f, 3.14f, 322.1337f, 1000.00f, 65545.17111337f};
    //Double[] array = {-10000.01, -6500.1234, -100.00001, -1.0, 0.00, -0.0, 1.0000, 1.1, 3.14, 3.14159265358979323846, 322.1337, 1000.00, 65545.17111337};
    for (Float f : array) {
      byte[] result = ByteBuffer
        .allocate(5)
        .order(ByteOrder.BIG_ENDIAN)
        .put(FLOAT_CODE)
        //.put(DOUBLE_CODE)
        .putFloat(f)
        //.putDouble(f)
        .array();
      floatingPointCoding(result, 1, true);
      System.out.println(f);
      System.out.println(Arrays.toString(result));
    }
    
    // System.out.println(Arrays.toString(result));
    // System.out.println(bytesToHex(result));
    // floatingPointCoding(result, 1, true);
    // System.out.println(bytesToHex(result));
  }
}
```